antarest/launcher/web.py

```diff
@@ -203,14 +205,44 @@ def get_load(
     @bp.get(
         "/launcher/_versions",
         tags=[APITag.launcher],
-        summary="Get list of supported study version for all configured launchers",
-        response_model=Dict[str, List[str]],
+        summary="Get list of supported solver versions",
+        response_model=List[str],
```

Comment:
a detail but you can add the `status_code = 200`

Reply:
Not needed, because [200 is the default status code](https://fastapi.tiangolo.com/tutorial/response-status-code/#about-http-status-codes).

> the more you have lines, the more you have bugs 😋